pkg/translator/jaeger/jaegerproto_to_traces.go

```diff
@@ -258,22 +258,24 @@ func jTagsToInternalAttributes(tags []model.KeyValue, dest pcommon.Map) {
 	}
 }
 
-func setInternalSpanStatus(attrs pcommon.Map, dest ptrace.SpanStatus) {
+func setInternalSpanStatus(attrs pcommon.Map, span ptrace.Span) {
+	dest := span.Status()
```

Comment:
Why changing the signature?

Reply:
As far as i understood the spec, we doint set en error based on the http status code if the span kind is server.
`For HTTP status codes in the 4xx range span status MUST be left unset in case of SpanKind.SERVER and MUST be set to Error in case of SpanKind.CLIENT.`

So we do need to ignore things here, right?
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12753/files#diff-79ad71b7ff1af223e5f80a4d1865ea75dc8db830e0ffbda369be65d104fe9a65R311

Reply:
Makes sense now. Added another commend related to that